prestoadmin/server.py

```diff
@@ -375,16 +375,23 @@ def install(rpm_specifier):
                         should ignore checking Presto rpm package
                         dependencies. Equivalent to adding --nodeps
                         flag to rpm -i.
+
+        --no-config -   pass this in order to avoid server config after installation
```

Comment:
did you mean to `avoid update`? Maybe switch should be named `--no-config-update`?

Reply:
works for me - renaming switch and updating comment